maths/3n+1.py

```diff
@@ -15,10 +21,10 @@ def n31(a):
             a = 3*a +1
         counter += 1
         path += [a]
-    return path , counter
+    return path, counter + 1
```

Comment:
Would it also work to __return path, len(path)__ ?

Reply:
Yes, that is more elegant, thanks!